inference-engine/tests/functional/shared_test_classes/src/single_layer/conversion.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/conversion.hpp"
```

Comment:
You have added new `conversion.cpp` file but the old files still exist, try to remove:
[/inference-engine/tests/functional/shared_test_classes/src/single_layer/convert_like.cpp](https://github.com/openvinotoolkit/openvino/blob/b48c52a1afb355e95a2c61ae8ee490cd8f9c3872/inference-engine/tests/functional/shared_test_classes/src/single_layer/convert_like.cpp)
and
[/inference-engine/tests/functional/shared_test_classes/src/single_layer/convert.cpp](https://github.com/openvinotoolkit/openvino/blob/b48c52a1afb355e95a2c61ae8ee490cd8f9c3872/inference-engine/tests/functional/shared_test_classes/src/single_layer/convert.cpp)

Reply:
Done! All conversion test have been updated.